docs/src/inputs/dropdownDeprecated/dropdownDeprecated.jsx

```diff
@@ -7,1833 +7,1701 @@
-                        </Header>
-
-                        <Dropdown
-                            onChange={this._onSelectionCustomOptionsChange.bind(this)}
-                            selectionOptionComponent={SelectionCustomComponent}
-                            options={selectionCustomOptions}
-                            placeholder="Select a user"
-                            selection
-                            value={this.state.selectionCustomOptionsValue}
-                        />
-
-                        <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
-                            {selectionOptionComponentSample}
-                        </Highlighter>
-
-                        {/* Selection Underline */}
-                        <Header anchor="selection-underline" size="large" style={{ marginTop: '55px' }} sub>
-                            Selection Underline
-                            <Header.Subheader>
-                                A Dropdown selection can be styled to have an underline.
-                            </Header.Subheader>
-                        </Header>
-
-                        <Dropdown
-                            onChange={this._onSelectionCustomOptionsChange.bind(this)}
-                            options={selectionCustomOptions}
-                            placeholder="Select a user"
-                            selection
-                            selectionUnderline
-                            value={this.state.selectionCustomOptionsValue}
-                        />
-
-                        <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
-                            {selectionUnderlineSample}
-                        </Highlighter>
-
-                        {/* Theme */}
-                        <Header anchor="theme" size="large" style={{ marginTop: '55px' }} sub>
-                            Theme
-                            <Header.Subheader>
-                                Changes the menu's color.
-                            </Header.Subheader>
-                        </Header>
-
-                        <Dropdown button placeholder="Button Dropdown" theme="dark">
-                            <Dropdown.Item label="Option 1" />
-                            <Dropdown.Item label="Option 2" />
-                            <Dropdown.Item label="Option 3" />
-                            <Dropdown.Item label="Option 4" />
-                        </Dropdown>
-
-                        <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
-                            {themeSample}
-                        </Highlighter>
-                    </div>
-                </Main.Content>
-            </Main>
-        );
-    }
-}
-
-export default ModulesDropdown;
+ import 'images/avatar1.jpg';
+ import 'images/avatar2.jpg';
+ import 'images/avatar3.jpg';
+ import 'images/avatar4.jpg';
+ import 'images/avatar5.jpg';
```

Comment:
Did we mess with the whitespace here? 😕 
Can we revert that?

Reply:
VS Code has been doing something wierd when I copy and past giving me one space of intention to much. Sorry about that.

Reply:
fixed.

Reply:
Looks better now.  Sorry about VSCode being a pain. :-(
Resolving.